server/views/manage/outOfServiceBeds/cancel.njk

```diff
@@ -0,0 +1,42 @@
+{% from "../../partials/showErrorSummary.njk" import showErrorSummary %}
+{% from "moj/components/interruption-card/macro.njk" import interruptionCard %}
+{% extends "../../partials/layout.njk" %}
+
+{% set pageTitle = applicationName + " - " + pageHeading %}
+{% set mainClasses = "app-container govuk-body" %}
+
+{% block content %}
+<div class="govuk-grid-row">
+    <div>
+
```

Comment:
These seem unnecessary -- and one of them is unclosed? The content is overflowing -- probably as a result of these.